src/main/java/org/opensearch/geospatial/search/aggregations/bucket/geogrid/GeoHexHelper.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.search.aggregations.bucket.geogrid;
+
+import static org.opensearch.geospatial.h3.H3.MAX_H3_RES;
+import static org.opensearch.geospatial.h3.H3.MIN_H3_RES;
+import static org.opensearch.geospatial.h3.H3.geoToH3;
+import static org.opensearch.geospatial.h3.H3.h3IsValid;
+import static org.opensearch.geospatial.h3.H3.h3ToLatLng;
+import static org.opensearch.geospatial.h3.H3.stringToH3;
+
+import java.util.Locale;
+
+import lombok.NonNull;
+
+import org.opensearch.common.geo.GeoPoint;
+
+/**
+ * Helper class for H3 library
+ */
+public class GeoHexHelper {
```

Comment:
just want to make sure that in this repo we're using Helper class suffix. I think in k-NN it's usually Util, but I finer with either one as long as we're consistent

Reply:
Right. We are calling it a helper than a utility everywhere.